project/build.scala

```diff
@@ -8,12 +8,12 @@ import sbtunidoc.Plugin._
 import ScalaJSPlugin.autoImport._
 
 object BuildSettings {
-  import Publishing._
+
   implicit val logger = ConsoleLogger()
 
-  val buildSettings = publicationSettings ++ defaultScalariformSettings ++ Seq(
+  val buildSettings = Publishing.publicationSettings ++ defaultScalariformSettings ++ Seq(
     organization := "org.w3",
-    version := "0.8.0-SNAPSHOT",
+    version := "0.8.0." + GitHelper.headSha(),
```

Comment:
> does that mean that all the jars are now with an sha1 sum?

Correct.

> Does that mean that you need to change the code that uses this to use the sha1 sum instead if you are publishing locally?

Correct again. SNAPSHOT by experience is actually painful and error prone (did I invalidate all my caches?). And it's not compatible with bintray.